src/core/router/readme.md

```
@@ -0,0 +1,12 @@
+# Router
+
+The router is responsible for building a set of available routes based on feature flags and user state. Each route is described in `routes.tsx` with an optional feature flag requirement. During application startup the router performs an initial redirect to make sure the user lands on a meaningful page.
+
+### Initial navigation logic
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)